### PR TITLE
Fix/spawn async

### PR DESCRIPTION
**Pull request by qnikst**
Extend test-suite with test for [already merged PR](https://github.com/haskell-distributed/distributed-process/pull/195)

### CODE REVIEW

**Comment by qnikst** [comment]
@facundominguez please check this PR, I've extended commit message and rebased it on the top of the current work.

File: src/Control/Distributed/Process/Tests/CH.hs [hunk 1]
```diff
@@ -719,6 +719,22 @@ testSpawnLocal TestTransport{..} = do
...
+testSpawnAsyncStrictness :: TestTransport -> Assertion
+testSpawnAsyncStrictness TestTransport{..} = do
+  node <- newLocalNode testTransport initRemoteTable
+  done <- newEmptyMVar
+
+  runProcess node $ do
+    here <-getSelfNode
+
+    ev <- try $ spawnAsync here (error "boom")
+    liftIO $ putStrLn "here"
```
Comment (facundominguez) [inline src/Control/Distributed/Process/Tests/CH.hs]: Remove or make it more meaningful.

**Comment by facundominguez** [comment]
:+1:

Maybe d-p should have a test suite for internal functions like `sendCtrlMsg`.

**Comment by qnikst** [comment]
I agree with possible introduction of test in d-p but I'd like to discuss what should be there before.

**Pull request merged**